src/azure-cli-core/setup.py

```diff
@@ -53,16 +53,13 @@
     'jmespath',
     'knack~=0.8.2',
     'msal>=1.10.0,<2.0.0',
-    # Dependencies of the vendored subscription SDK
-    # https://github.com/Azure/azure-sdk-for-python/blob/ab12b048ddf676fe0ccec16b2167117f0609700d/sdk/resources/azure-mgmt-resource/setup.py#L82-L86
-    'msrest>=0.5.0',
-    'msrestazure>=0.6.3',
     'paramiko>=2.0.8,<3.0.0',
     'pkginfo>=1.5.0.1',
     'PyJWT==1.7.1',
     'pyopenssl>=17.1.0',  # https://github.com/pyca/pyopenssl/pull/612
     'requests~=2.22',
     'six~=1.12',
+    'urllib3[secure]>=1.26.4',
```

Comment:
The dependency on `urllib3` should go to `azure-cli-core` as `requests` is required here.

https://github.com/Azure/azure-cli/blob/f50cf6b3d63f4cdf45afc783c172394f2b990fd0/src/azure-cli-core/setup.py#L60